### ISSUE TITLE
Disabled in TB 78.8.1

### ISSUE THREAD

**Issue by gigjozsa** [kewisch/gdata-provider#181]
### Describe the problem and steps to reproduce it:
Provider for google calendar is disabled in TB 78.8.1
(Please include as many details as possible.)

### What happened?
After an automatic update of TB to 78.8.1 in Ubuntu 20.04 (1:78.8.1+build1-0ubuntu0.20.04.1), the addon (68.2.1) gets disabled because it is declared incompatible "Provider for Google Calendar is incompatible with Thunderbird 78.8.1.". I tend to use the packaged versions, and TB 78.8.1 is the latest that currently installs.

### What did you expect to happen?
Before the upgrade of TB the addon was running fine.

### Anything else we should know?
This has also been reported by someone else in the [Reviews on the addon page](https://addons.thunderbird.net/en-US/thunderbird/addon/provider-for-google-calendar/).

**Comment by MLopez-Ibanez** [comment]
I'm running Provider for Google Calendar 78.0.1 in Ubuntu 20.04 and it is enabled. You need to update add-ons

**Comment by gigjozsa** [comment]
@MLopez-Ibanez Thanks a lot. It is working now for me as well, but I did not do a thing to achieve this. TB automatic addon updating was switched on (also at the time I posted the issue), and the current version of the addon is 78.0.1 (was 68.2.1 two days ago, see above). I guess the update frequency of TB is lower than I expected, and I did not find out how to update manually (just found out now that you actually have to switch off automatic updating to get a button saying "update now".
